### ISSUE TITLE
Blockquote footer color contrast issue

### ISSUE THREAD

**Issue by peterserfozo** [openeuropa/bootstrap-component-library#707]
During an accessibility audit, a color contrast issue was identified with the blockquote text. We should improve the contrast by darkening the text color relative to the background.

```Element has insufficient color contrast of 4.44 (foreground color: #6c757d, background color: #f8f9fa, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1```

```
<figcaption class="blockquote-footer m-0">...</figcaption>
```

<img width="1404" height="219" alt="Image" src="https://github.com/user-attachments/assets/198aa8b7-245f-4a4c-98c4-8b181819b322" />

See https://github.com/openeuropa/bootstrap-component-library/blob/development/src/themes/default/src/scss/_blockquote.scss#L3

**Comment by tibi2303** [comment]
<img width="1734" height="818" alt="Image" src="https://github.com/user-attachments/assets/bf19d37b-0cd8-468e-8dd7-995256860d1c" />


On theme default the text muted is #495057 which should pass the accessibility rule.

<img width="727" height="566" alt="Image" src="https://github.com/user-attachments/assets/04b47d0e-e4ac-4dd8-9596-d8ef9496480b" />

The color you have is the default bootstrap one but was overwritten by us at default theme level (the file url you attached). 

Is it still the case for this issue?
